.github/workflows/pr-comment.yml

```diff
@@ -3,16 +3,13 @@ on:
   issue_comment:                                     
     types: [created, edited, deleted]
 jobs:
- job01:
-   if: contains(github.event.comment.html_url, '/pull/')    # check if the comments come from pull request, exclude those from issue.
-   runs-on: [re-centos7]
-   steps:
-    - name: say hello
-      if: contains(github.event.comment.body, '/support')   # check the comment if it contains the keywords
-      run: |
-       echo say hello
-    - name: test
-      if: contains(github.event.comment.body, '/test')  # check the comment if it contains the keywords
-      run: |
-       echo github.event.comment.body
```

Comment:
That's retarded. Brilliant, Rails, the pundit of less is more, now requires an if statement around a field because Rails is too retarded to pluralize the word cafe. How about you change the name of the function to pluralize_some_stuff, it'd define the function a lot better.